Mathmagician/Prime.cs

```diff
@@ -18,10 +18,51 @@ public int FollowingPrime(int v)
             return 3;
         }
 
-        public List<int> Sequence(int v)
+ /*       public static bool isPrime(int increasingNumber)

```

Comment:
Be sure to clean this up before the project is done. (I know you might need it right now)